staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -355,52 +387,94 @@ func strategicPatchObject(
 
 // applyPatch is called every time GuaranteedUpdate asks for the updated object,
 // and is given the currently persisted object as input.
-func (p *patcher) applyPatch(_ context.Context, _, currentObject runtime.Object) (runtime.Object, error) {
+// TODO: rename this function because the name implies it is related to applyPatcher
+func (p *patcher) applyPatch(_ context.Context, _, currentObject runtime.Object) (objToUpdate runtime.Object, patchErr error) {
```

Comment:
Nit: I personally think this code is less clear for having the named return args (and it doesn't look like we're doing any defer magic that would require it)